makefile

```diff
@@ -423,31 +423,10 @@ docker_buildandpackage: docker_ok
 # Test targets        #
 #---------------------#
 
-ifndef NORACETEST
-ifeq (,$(findstring history_size,$(GORACE)))
-export GORACE:=$(GORACE) history_size=7
-endif
-ifeq (,$(findstring halt_on_error,$(GORACE)))
-export GORACE:=$(GORACE) halt_on_error=1
-endif
-ifeq (,$(findstring -race,$(GOTEST_FLAGS)))
-export GOTEST_FLAGS:=$(GOTEST_FLAGS) -race
-endif
-endif
```

Comment:
I was thinking that, but since the user can as easily run go test ./... with whatever flags they want, I leaned towards making this script simpler.